tutorials/hello-world/hello-world.md

```diff
@@ -148,7 +201,11 @@ int main(int argc, char *argv[]) {
 }
 /*- endfilter --*/
 ```
-Once you have made your change, rerun `ninja` to rebuild the project:
+Once you have made your change, use Terminal B to rebuild the project.
+
+*Hint:* Remember to exit the QEMU siumator before rerunning the project with `ctrl-A,x`.
+
+Then rebuild using ninja and run the simulator again:
```

Comment:
Was this one supposed to replace the next line?

Reply:
I've removed line 208